components/brave_shields/browser/ad_block_service.h

```diff
@@ -116,6 +119,7 @@ class AdBlockService {
   AdBlockRegionalServiceManager* regional_service_manager();
   AdBlockSubscriptionServiceManager* subscription_service_manager();
   AdBlockCustomFiltersProvider* custom_filters_provider();
+  AdBlockLocalhostFiltersProvider* localhost_filters_provider();
```

Comment:
this shouldn't be needed - the others only have accessors because the UI needs to access them for other purposes